package.json

```diff
@@ -74,8 +74,8 @@
     "ag-grid-community": "28.1.1",
     "ag-grid-react": "28.1.1",
     "i18next": "~21.8.9",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
```

Comment:
Is this change intended? We remove the pinning and let yarn grab the latest minor / patch?